ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -1287,6 +1287,21 @@ def serialize_module(
                             trigger_class = inputs_obj.workflow_trigger
                             if trigger_class is not None:
                                 row_data["workflow_trigger_id"] = str(trigger_class.__id__)
+                            if inputs_obj.node_output_mocks is not None:
+                                serializer_context = {
+                                    "serializer": lambda value: serialize_value(
+                                        workflow_display.workflow_id,
+                                        workflow_display.display_context,
+                                        value,
+                                    )
+                                }
+                                serialized_mocks = inputs_obj.model_dump(
+                                    mode="json",
+                                    include={"node_output_mocks"},
+                                    context=serializer_context,
+                                )
+                                if "node_output_mocks" in serialized_mocks:
+                                    row_data["node_output_mocks"] = serialized_mocks["node_output_mocks"]
```

Comment:
DevinAI, we should be able to replace this all with a single `inputs_obj.model_dump`